shared/src/state.ts

```diff
@@ -48,6 +48,7 @@ export class ExerciseState {
     public readonly tileMapProperties: TileMapProperties =
         defaultTileMapProperties;
     public readonly statistics: readonly StatisticsEntry[] = [];
+    public readonly pretriageEnabledFlag: boolean = true;
```

Comment:
I would rather do it like this:

```ts
public readonly configuration: ExerciseConfiguration;
```

And:

```ts
class ExerciseConfiguration {
    public readonly pretriageIsEnabled: boolean = true;
}
```

Reply:
When I read `...Flag` I never know whether this means enabled or disabled. Is there a clear standard for that?